servicetalk-client-api/src/main/java/io/servicetalk/client/api/AbstractRetryingFilterBuilder.java

```diff
@@ -287,7 +294,7 @@ private ReadOnlyRetryableSettings(final int maxRetries,
             this.initialDelay = initialDelay;
             this.timerExecutor = timerExecutor;
             this.exponential = exponential;
-            this.jitter = jitter;
+            this.jitter = requireNonNull(jitter);
```

Comment:
Perhaps range check here rather than in Strategies?

Reply:
I would prefer to keep the value validation in a single spot to avoid duplicating logic.